messages_appendix.cddl

```diff
@@ -75,23 +75,24 @@ auth-initiation-token = (
   ? 0 : text ; token
 )
 
-; type key 1002
-auth-spake2-need-psk = {
-  auth-initiation-token
-}
+auth-spake2-psk-status = &(
+ psk-needs-presentation: 0
+ psk-shown: 1
+ psk-input: 2
+)
 
-; type key 1003
+; type key 1002
 auth-spake2-handshake = {
-  auth-initation-token
-  1 : bytes .size 32 ; random
+  0: auth-spake2-psk-status ; psk-status
+  1: bytes ; public-value
 }
 
-; type key 1004
+; type key 1003
 auth-spake2-confirmation = {
-  1 : bytes .size 32 ; transcript-mac
+  1 : bytes .size 64 ; transcript-mac
 }
 
-; type key 1005
+; type key 1004
 auth-status = {
   1 : auth-status-result ; result
```

Comment:
Where is auth-status-result defined? It's not in this CDDL file?

Reply:
It's defined below, I moved it up right before where it is used here.